src/fideslang/models.py

```diff
@@ -418,6 +425,10 @@ class FidesMeta(BaseModel):
         default=None,
         description="Optionally specify that a field may be used as a custom request field in DSRs. The value is the name of the field in the DSR.",
     )
+    masking_strategy_override: Optional[FieldMaskingStrategyOverride] = Field(
```

Comment:
Can we name this `field_masking_strategy_override` to avoid confusion with the collection-level strategy?